pandas/tests/indexes/common.py

```diff
@@ -721,3 +736,18 @@ def test_fillna(self):
                 expected[1] = True
                 self.assert_numpy_array_equal(idx._isnan, expected)
                 self.assertTrue(idx.hasnans)
+
+    def test_copy(self):
```

Comment:
IIRC ``test_identical`` should be in ``Base`` I think. Just want to have the tests grouped a bit more (e.g. the copy-related ones together)